pkg/reconciler/dependencybuild/scripts/gradle-build.sh

```diff
@@ -34,13 +34,20 @@ export LC_ALL="en_US.UTF-8"
 ADDITIONAL_ARGS=$(echo "$@" | sed 's/build publish \?//')
 echo ADDITIONAL_ARGS="${ADDITIONAL_ARGS}"
 
+INIT_SCRIPTS=""
+for i in .hacbs-init/*
+do
+  INIT_SCRIPTS="$INIT_SCRIPTS -I $i"
```

Comment:
Do we want to number the scripts so that there's a set order to them in case there is more than one?

Reply:
The shell expansion will do them alphabetically anyway, so if we start adding more we can order them in the code that actually copies them to disk.